src/modules/Packages.rb

```diff
@@ -429,86 +426,9 @@ def CheckDiskSize(init)
     # Checks which products have been selected for removal and modifies
     # the warning messages accordingly.
     #
-    # @param [Yast::ArgRef] ret reference to map MakeProposal->Summary
-    def CheckOldAddOns(ret)
-      products = Pkg.ResolvableProperties("", :product, "")
```

Comment:
Just a question: why we are dropping this?

Reply:
We have talked together about it some weeks ago :-)
Currently we do not have any workflow for that.